### PR TITLE
refactor code

### PR DESCRIPTION
**Pull request by gauravchawla73463**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/ECM-665

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.OLD_DATE_TIME_PATTERN;
+import static uk.gov.hmcts.ethos.replacement.docmosis.reports.memberdays.MemberDaysReport.OLD_DATE_TIME_PATTERN3;
+
+public class ReportCommonMethods {
+
+    private ReportCommonMethods() {
+    }
+
+    public static String getHearingDuration(DateListedTypeItem c) {
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java:16]: Duration in what? Minutes? Hours? Either you code has to be self-documenting or you provide a javadoc

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.reports;
+
+import com.microsoft.azure.servicebus.primitives.StringUtil;
```
Comment (jthmcts) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java:3]: wrong class imported

Comment (hsjhita) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java:3]: library used and maybe import isNullOrEmpty only

Comment (hsjhita) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java:3]: resolving as James has mentioned

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java (hunk 1 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java]: refactored/renamed method name

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java (hunk 2 above)
Comment (gauravchawla73463) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/ReportCommonMethods.java]: refactored

**Review by jthmcts — APPROVED** [review]

**Review by hsjhita — APPROVED** [review]

**Pull request merged**